### PR TITLE
updated alert border colors

### PR DESCRIPTION
**Pull request by pweikel**

### CODE REVIEW

**Review by danthewolfe — APPROVED** [review]

File: src/components/Contentful/Alert/style.module.css [hunk 1]
```diff
@@ -8,13 +8,23 @@
...
+}
+.alertSection.warning {
+  border: solid 2px #ad5458;
+}
+.alertSection.serviceAvailable {
+  border: solid 2px #1d7a4f;
+}
+.alertSection.serviceLimited {
+  border: solid 2px #1d7a4f;
+}
```
Comment (mlkirkpatrick) [inline src/components/Contentful/Alert/style.module.css:25]: Combine rules that use the same styles like so:
```
.alertSection.serviceAvailable,
.alertSection.serviceLimited {
  border: solid 2px #1d7a4f;
}
```

File: src/components/Contentful/Alert/style.module.css [hunk 2]
```diff
@@ -8,13 +8,23 @@
...
 .alertSection.serviceUnavailable {
-  border: solid 2px #AA272F;
+  border: solid 2px #1d7a4f;
 }
```
Comment (mlkirkpatrick) [inline src/components/Contentful/Alert/style.module.css:28]: This isn't in use, but if it was, it should use the same color as warning. Suggest combining those two.

**Review by mlkirkpatrick — APPROVED** [review]
Approved, with minor suggested changes

File: src/components/Contentful/Alert/style.module.css (hunk 1 above)
Comment (pweikel) [reply src/components/Contentful/Alert/style.module.css]: Condensed CSS

File: src/components/Contentful/Alert/style.module.css (hunk 2 above)
Comment (pweikel) [reply src/components/Contentful/Alert/style.module.css]: Combined

**Pull request merged**